SU-HSDO/suhumsci
# STN-603 : Carousel Controls Slick Dots and Arrows Refinement

# READY FOR REVIEW

## Summary
Refine the appearance of the carousel controls (slick dots and arrows). This work is for both the Traditional and Colorful themes. It also includes carousels with content and those without any content which feature only an image.

Reference: [JIRA STN-603](https://sparkbox.atlassian.net/browse/STN-603)

## Need Review By (Date)
ASAP

## Urgency
high

## Steps to Test
1. In your CLI, run `npm run test` and confirm our tests pass.
2. The staging environment has been updated with pages for testing this PR. You can sync with staging or follow the instructions provided in the steps below to test with the content on your local environment.
3. Review controls on carousels without content that feature an image only. Go to http://sparkbox-sandbox.suhumsci.loc/qa/qa-carousel-images
- If you did not sync with staging, create a basic (flexible) page and add a carousel hero image with multiple slides which only have an image. Do not add any content.
4. Review the controls on carousels with content. Go to http://sparkbox-sandbox.suhumsci.loc/qa/carousel-hero-image
- In you did not sync with staging, In order to review the carousel controls you need to edit the hero carousel and add additional slides with content.
5. Review carousel controls on the dev environment to confirm the adjustments made to the SVGs and their associated styles display. https://sparkbox-sandbox-dev.stanford.edu/qa-carousel-images
